server/client/profile/profile.css

```diff
@@ -105,7 +172,7 @@ p {
     }
 
     .profile-container {
-        width: 75%;
+        width: 90%;
```

Comment:
As Cris said, try to avoid using %. If you really need it, use only 0%, 25%, 50%, 75% or 100%

Reply:
updated to 75%